roles/mailserver/files/docker-compose.yml

```diff
@@ -165,6 +194,13 @@ services:
       - mailserver
       - postfixadmin
       - rainloop
+{% if use_poweradmin %}
+      - poweradmin
+{% endif %}
```

Comment:
Why is this repeated?

Reply:
repeated ?? i followed the same logic user for opencart

Reply:
Look at the `nginx` service definition in your branch's docker-compose. Looks like some overzealous copy-pasting going on. The `depends_on` section is being repeated multiple times there.

```yaml
  # Web server
  # https://github.com/Wonderfall/dockerfiles/tree/master/nginx
  # https://nginx.org/
  # Configuration : https://github.com/hardware/mailserver/wiki/Reverse-proxy-configuration
  nginx:
    image: wonderfall/boring-nginx
    container_name: nginx
    restart: on-failure
    ports:
      - "80:8000"
      - "443:4430"
    volumes:
      - /mnt/docker/nginx/sites-enabled:/sites-enabled
      - /mnt/docker/nginx/conf:/conf.d
      - /mnt/docker/nginx/log:/var/log/nginx
{% if self_signed_certs %}
      - /mnt/docker/mail/ssl/selfsigned:/certs
{% else %}
      - /mnt/docker/nginx/certs:/certs
{% endif %}
      - /mnt/docker/www:/www
{% if www_is_wordpress %}
      - /mnt/docker/wordpress:/wordpress
{% endif %}
    depends_on:
      - mailserver
      - postfixadmin
      - rainloop
{% if use_opencart %}
      - opencart
{% endif %}
    depends_on:
      - mailserver
      - postfixadmin
      - rainloop
{% if use_poweradmin %}
      - poweradmin
{% endif %}
    depends_on:
      - mailserver
      - postfixadmin
      - rainloop
{% if www_is_wordpress %}
      - wordpress
{% else %}
      - contact
{% endif %}
```